python/dgl/data/qm9_edge.py

```diff
@@ -0,0 +1,404 @@
+"""QM9 dataset for graph property prediction (regression)."""
+"""When building this class, we partially refer to the implementation from https://github.com/Jack-XHP/DGL_QM9EDGE """
+
+import os
+import numpy as np
+
+from .dgl_dataset import DGLDataset
+from .utils import download, extract_archive, _get_dgl_url
+from ..convert import graph as dgl_graph
+from .. import backend as F
+
+try:
+    import rdkit
```

Comment:
@jermainewang Are you fine with having RDKit as an optional dependency when importing this dataset?